.genx/scripts/update-versions.js

```diff
@@ -20,8 +20,8 @@ const writeJSON = (json, path) => {
 };
 
 const UI = run('npm info @genesislcap/foundation-ui@latest version');
-const GSF = run(`jf rt s "libs-release-client/global/genesis/genesis-distribution/" --sort-by="name;created" --sort-order=desc --limit=1 --exclusions="*-RC*;*-SNAPSHOT*;*maven-metadata*;*test*;*TEST*" | grep path | tr -s ' ' | cut -d '/' -f 5`);
```

Comment:
We can't be sure that jfrog will return us the correct version when limiting to 1, so we find latest version from all the published versions